kornia/testing/__init__.py

```diff
@@ -246,10 +246,22 @@ def _get_precision_by_name(
     return tol_val_default
 
 
+# {dtype: (rtol, atol)}
+_DTYPE_PRECISIONS = {
+    torch.float16: (1e-3, 1e-3),
+    torch.float32: (1e-4, 1e-5),
+    torch.float64: (1e-5, 1e-8),
+}
+
+
+def _default_tolerances(*tensors: torch.Tensor) -> Tuple[float, float]:
```

Comment:
I've vendored a minimal version of this so we don't need to import from a private namespace and thus should be safe on all `torch` versions.